dev/README.md

```diff
@@ -8,15 +8,19 @@ The server is managed by a third-party library called [`brokerapi`](https://gith
 
 ## Testing
 
-Please refer to the how-to documentation [action](https://github.com/mongodb/atlas-osb/blob/master/.github/HOWTO.md)
+We are using GitHub Action for testing purposes.
+For testing in CF used: `.github/workflows/deploy-broker.yml`
+For testing in K8S: `.github/workflows/k8s-demo-*.yml`
+Please refer to the [HOWTO.md](https://github.com/mongodb/atlas-osb/blob/master/.github/HOWTO.md) documentation
 
 ## Releasing
 
-The release process consists of publishing a new Github release with attached binaries as well as publishing a Docker image to [quay.io](https://quay.io). Evergreen can automatically build and publish the artifacts based on a tagged commit. To perform a release the [Evergreen CLI](https://github.com/evergreen-ci/evergreen/wiki/Using-the-Command-Line-Tool) needs to be installed.
+The release process consists of publishing a new Github release with attached binaries as well as publishing a Docker image to [quay.io](https://quay.io). Evergreen can automatically build and publish the artifacts based on a tagged commit.
```

Comment:
[nit] sounds like the "Evergreen..." sentence is out-of-date?

Reply:
You're right. deleted